test/unit/errors/activeUser.test.js

```diff
@@ -53,12 +53,25 @@ describe('ActiveUserError', () => {
   describe('code', () => {
     it('should return the default code -1', () => {
       const error = new ActiveUserError();
-      expect(error.debug).toEqual('');
+      expect(error.code).toEqual(-1);
     });
 
     it('should return the custom code 309', () => {
       const error = new ActiveUserError(undefined, undefined, 309);
       expect(error.code).toEqual(309);
     });
   });
+
+  describe('kinveyRequestId', () => {
+    it('should return undefined as the default value', () => {
+      const error = new ActiveUserError();
+      expect(error.kinveyRequestId).toEqual(undefined);
+    });
+
+    it('should return 1', () => {
+      const kinveyRequestId = 1;
+      const error = new ActiveUserError(null, null, null, kinveyRequestId);
```

Comment:
I think a more practical way to test this is to try a request with a mock, and then test that the active user error gets thrown with the correct request ID. Our error test cases should actually test for the correct errors being thrown.